File: CoreJava/basic-app/src/com/excerise/student/Dry.java
package com.excerise.student;

import java.util.*;

public class Dry {
	private String username;
	private String password;
	ArrayList<String> User = new ArrayList<String>();

	public String getUsername() {
		return username;
	}

	public void setUsername(String username) {
		this.username = username;
	}

	public String getPassword() {
		return password;
	}

	public void setPassword(String password) {
		this.password = password;
	}

	public String emaillogin() {
		addUser(getUsername(), getPassword());
		return "You are login through Email LOGIN Method";
	}

	public String fblogin() {
		addUser(getUsername(), getPassword());
		return "You are login through FB LOGIN Method";
	}

	public void addUser(String username, String password) {
		User.add(username);
	}

	public void showUser() {
		for (int i = 0; i < User.size(); i++) {
			System.out.println("Username: " + User.get(i));
		}
	}
}

File: DotNET/C#/BasicDryApp/BasicDryApp/Dry.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace BasicDryApp
{
    class Dry
    {
        private String _username;
        private String _password;
        private List<String> _user = new List<string>();

        public String Username
        {
            get
            {
                return _username;
            }
            set
            {
                _username = value;
            }
        }

        public String Password
        {
            get
            {
                return _password;
            }
            set
            {
                _password = value;
            }
        }

        public void AddUser(String username)
        {
            _user.Add(_username);
        }

        public void ShowUser()
        {
            foreach (String user in _user)
            {
                Console.WriteLine(user);
            }
        }

        public String emaillogin()
        {
            AddUser(Username);
            return "You are login through Email LOGIN Method";
        }

        public String fblogin()
        {
            AddUser(Username);
            return "You are login through FB LOGIN Method";
        }
    }
}
